Ensure we can compile with more recent versions of autotools.

Index: pmacct-1.5.3/configure.in
===================================================================
--- pmacct-1.5.3.orig/configure.in
+++ pmacct-1.5.3/configure.in
@@ -2,7 +2,7 @@ dnl Process this file with autoconf to p
 dnl configuration file for pmacct 
 
 AC_INIT([src/pmacctd.c], [1.5.3], [paolo@pmacct.net])
-AM_INIT_AUTOMAKE([pmacctd], [1.5.3])
+AM_INIT_AUTOMAKE([foreign])
 AC_PREFIX_DEFAULT([/usr/local])
 
 COMPILE_ARGS="${ac_configure_args}"
